src/librvnpal/src/win/mapping.c

```diff
@@ -16,20 +16,37 @@ _init_discard_virtual_memory_flag = false;
 EXPORT int32_t
 rvn_discard_virtual_memory(void* address, int64_t size, int32_t* detailed_error_code)
 {    
+    int32_t rc;
     if (_init_discard_virtual_memory_flag == false)
     {        
+        HMODULE handle = GetModuleHandle(TEXT("kernel32.dll"));
+        if(handle == NULL)
```

Comment:
_can_ this fail?

Reply:
@haludi ?

Reply:
No, this can't happen and the check here is for satisfying the Code Analysis